imap_data_access/cli.py

```diff
@@ -277,6 +305,9 @@ def main():  # noqa: PLR0915
         choices=["table", "json"],
         default="table",
     )
+    query_parser.add_argument(
+        "--filename", type=str, required=False, help="Name of a file to be searched for"
```

Comment:
Not absolutely necessary, but might be nice to add a link to the naming convention documentation in the help string here, so users can find what the file naming convention is in case they don't know it:

https://imap-processing.readthedocs.io/en/latest/development-guide/style-guide/naming-conventions.html#data-product-file-naming-conventions